client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -138,20 +138,28 @@ class ListItem extends PureComponent {
     return action(_id, listId, name);
   };
 
-  handleConfirmationVisibility = event => {
+  hideConfirmation = event => {
     event.preventDefault();
 
-    this.setState(({ isConfirmationVisible }) => ({
-      isConfirmationVisible: !isConfirmationVisible
-    }));
+    this.setState({ isConfirmationVisible: false });
   };
 
-  handleMoveToListPanelVisibility = event => {
+  showConfirmation = event => {
     event.preventDefault();
```

Comment:
do we need this preventDeafult() calls? 
do we have any additional effects on those cancels, confirm buttons ??